src/Behavioral.Automation/Bindings/TableBinding.cs

```diff
@@ -20,9 +20,9 @@ public void CheckRowsCount([NotNull] ITableWrapper element, int count)
             Assert.ShouldBecome(() => element.Rows.Count(), count, $"{element.Caption} has {element.Rows.Count()} rows");
         }
 
-        [Given("(.*?) (contain|not contain) the following rows:")]
-        [Then("(.*?) should (contain|contain in exact order|not contain) the following rows:")]
-        public void CheckTableContainsRows(ITableWrapper gridRows, string behavior, Table table)
+        [Given("(.*?) (contain|not contain) the (following|only following) rows:")]
```

Comment:
Just as a suggestion. Maybe "the following (rows | rows only) will sound better. Don't you think?

Reply:
100% agree )